docs/workforce-management-framework/topics/pro-using-demo-app.adoc

```diff
@@ -0,0 +1,103 @@
+= Using the demo application user interface
+
+The {WFM-RC-LongName} demo application consists of three components:
+
+- A mobile application for end users
+- A portal application for administrator users on a web browser
+- A cloud application for backend server functionality (No user interface)
+
+The demo application has demo users and demo data to show off the functionality of {WFM-RC-LongName}.
+The demo application can define business process or procedure via a custome Workflow and assign these workflows as jobs to
+individual workers as Workorders. Workers can view their Workorder via a mobile applicaion and administrators
+can manage workorders via the web portal.
+
+== Portal application
+
+The Portal appliction is used by a administrator to view , edit and assign Workorder and to create new Workflows.
+The Portal application is a web application. The view of workorders is restricted to administrators. 
+Onced logged in you will be presented with the following screen for admistration of Workorders and Workflows.
+
+image::/shared/images/portal.png[portal]
+
+=== Create Workflow
+
+Click on Workflow in the left panel. To create a workflow click on the `CREATE WORKFLOW` button. This will bring you to the create workflow screen where you can add a title.
```

Comment:
That's instruction. It should relay on numbers.

> Click on Workflow in the left panel.

Select workflow *on* navigation panel